astroplan/constraints.py

```diff
@@ -506,15 +506,35 @@ def is_always_observable(constraints, observer, targets, times=None,
         List of booleans of same length as ``targets`` for whether or not each
         target is observable in the time range given the constraints.
     """
+    targets_is_scalar = False
+    times_is_scalar = False
+
+    if not hasattr(targets, '__len__'):
```

Comment:
Maybe I'm confused, but that doesn't seem to be happening. Here `non_scalar_target_sc` is a length=3 sky-coordinate object:
```python
In [14]: non_scalar_target_sc
Out[14]: 
<SkyCoord (ICRS): (ra, dec) in deg
    [(279.23473479, 38.78368896), (78.63446707, 8.20163837),
     (37.95456067, 89.26410897)]>

In [15]: len(non_scalar_target_sc)
Out[15]: 3

In [16]: non_scalar_target_sc.isscalar
Out[16]: False

In [17]: non_scalar_target_sc.__len__
Out[17]: 
<bound method SkyCoord.__len__ of <SkyCoord (ICRS): (ra, dec) in deg
    [(279.23473479, 38.78368896), (78.63446707, 8.20163837),
     (37.95456067, 89.26410897)]>>

In [18]: non_scalar_target_sc.__len__()
Out[18]: 3
```